simplekv/cache.py

```diff
@@ -110,6 +110,26 @@ def open(self, key):
             # cache error, ignore completely and return from backend
             return self._dstore.open(key)
 
+    def _copy(self, source, dest):
+        """Implementation of :meth:`~simplekv.CopyMoveMixin.copy`.
+        
+        Copies the data in the backing store and removes the destination key from the cache,
+         in case it was already populated.
+         """
+        k = self._dstore._copy(source, dest)
+        self.cache.delete(dest)
+        return k
+
+    def _move(self, source, dest):
+        """Implementation of :meth:`~simplekv.CopyMoveMixin.move`.
+        
+        Moves the data in the backing store and removes both the source and destination key from the cache.
+        """
+        k = self._dstore._move(source, dest)
+        self.cache.delete(source)
+        self.cache.delete(dest)
```

Comment:
Same as above, but here you will need two finally blocks to ensure the cache never gets stale under error conditions.